.git-blame-ignore-revs

```diff
@@ -0,0 +1,2 @@
+# Move queries to ./queries/scala sub-directory 
+c7e752ae212336b2b8f6eb7d9bc426434c384832
```

Comment:
I _think_ this is wrong. Shouldn't this be `889eb65aea8e0f9ce1694ab9ee528df1fa5f75d1`?

Reply:
Right, the rebase :+1: